src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -448,12 +515,20 @@ public sealed partial class PublisherClientImpl : PublisherClient
             );
 
         private readonly ClientHelper _clientHelper;
+        private readonly ApiCallableAsync<Topic, Topic> _createTopicAsync;
+        private readonly ApiCallableSync<Topic, Topic> _createTopicSync;
+        private readonly ApiCallableAsync<ListTopicsRequest, ListTopicsResponse> _listTopicsAsync;
+        private readonly ApiCallableSync<ListTopicsRequest, ListTopicsResponse> _listTopicsSync;
 
         public PublisherClientImpl(Publisher.IPublisherClient grpcClient, PublisherSettings settings)
         {
             this.GrpcClient = grpcClient;
             PublisherSettings effectiveSettings = settings ?? PublisherSettings.GetDefault();
             _clientHelper = new ClientHelper(effectiveSettings);
+            _createTopicAsync = _clientHelper.BuildApiCallableAsync(GrpcClient.CreateTopicAsync, effectiveSettings.CreateTopic_Retry);
```

Comment:
TODO: Think about whether it should be ApiCallableAsync or AsyncApiCallable. I don't think I've seen enough examples of asynchronous delegates to say either way.